sql/api/pom.xml

```diff
@@ -56,13 +56,6 @@
         <dependency>
             <groupId>org.json4s</groupId>
             <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
-            <version>3.7.0-M11</version>
```

Comment:
redundant, it's same as the one defined in root `pom.xml` `dependencyManagement`.

Reply:
Could you spin off this as a new PR, please, @pan3793 ? We can merge that before this PR.

Reply:
Okay

Reply:
opened https://github.com/apache/spark/pull/45943

Reply:
Thanks. I merged it.